src/EventProcessor.js

```diff
@@ -1,40 +1,45 @@
-var utils = require('./utils');
+import * as utils from './utils';
```

Comment:
The import changes are in a separate commit. The last commit just applies prettier formatting.